### PR TITLE
add initial version of RetailerCart

### PR DESCRIPTION
**Pull request by ayveezub**
image

### CODE REVIEW

**Review by ksevelyar — APPROVED** [review]
:shipit:

File: src/router/index.js [hunk 1]
```diff
@@ -1,10 +1,16 @@
 import { createRouter, createWebHistory } from 'vue-router'
 
+const RetailerCart = () => import('@/views/RetailerCart.vue')
```
Comment (ksevelyar) [inline src/router/index.js:5]: You need to rebase with the main branch

Comment (ayveezub) [reply src/router/index.js:5]: Done

**Review by dotnil — APPROVED** [review]
🤓

**Pull request merged**